packages/shared/src/features/onboarding/steps/InformativeScreen.tsx

```diff
@@ -0,0 +1,68 @@
+'use client';
+
+import React from 'react';
+import type { ReactElement } from 'react';
+import classNames from 'classnames';
+import StepHeadline from '../shared/StepHeadline';
+import { Image } from '../../../components/image/Image';
+import { Button, ButtonVariant } from '../../../components/buttons/Button';
+import type { FunnelStepFact } from '../types/funnel';
+
+type InformativeScreenProps = FunnelStepFact & {
+  onTransition: () => void;
+};
+
+const InformativeScreen = ({
+  parameters,
+  onTransition,
+}: InformativeScreenProps): ReactElement => {
+  return (
+    <>
+      {/* Background gradient */}
+      <div
+        className={
+          parameters?.reverse
+            ? 'bg-gradient-purple-reverse'
+            : 'bg-gradient-purple'
```

Comment:
These aren't our colors 🎃

Reply:
Do we have those colors in our palette somewhere? We need all those shadows for the funnel feature 👀 
<img width="708" alt="image" src="https://github.com/user-attachments/assets/ddcc260a-f812-4af2-a831-15217d869869" />

Reply:
Yeah, we used to have these colors. I'll check if I can recover what they used here in the past.

Reply:
Also, shouldn't this be inside the `content container`? That will make it easier to manage because the upper-level component could have a different set of properties. At least here, it would be predictable.

Reply:
Yeah they are just notions of the cabbage color I think :) 
Probably in design it uses the named ones already.